Electron-Basic-Screen-Sharing/README.md

```diff
@@ -0,0 +1,194 @@
+OpenTok.js Electron Basic Screen-Sharing Sample (Electron 17)
+=======================
+
+This sample application shows how to connect to an OpenTok session, publish a screen-sharing stream, and subscribe to a stream in a basic Electron application.
+
+## Quick started
+1. Get values for your OpenTok API key, session ID, and token. You can obtain these values from your TokBox account. Make sure that the token isn't expired.
+For testing, you can use a session ID and token generated at your TokBox account page. However, the final application should obtain these values using the [OpenTok server SDKs](https://tokbox.com/developer/sdks/server/). For more information, see the OpenTok developer guides on [session creation](https://tokbox.com/developer/guides/create-session/) and [token creation](https://tokbox.com/developer/guides/create-token/).
+
+2. `$ yarn install`
+
+3. `$ yarn start`
+
+*Important:* Read the following sections of the of the README file to get started
+
+## Installing Dependencies with `yarn`
+
+`$ yarn install`
```

Comment:
Again, I think we should use npm here to be consistent (and more make it slightly more accesible)

Reply:
That makes sense. Not everyone uses/installs yarn and we've been using npm for all of the other apps.